packages/common/src/store/premiumContent/reducer.ts

```diff
@@ -1,20 +1,20 @@
-import { ID, PremiumContentSignature } from '@audius/common'
+import { ID, PremiumContentSignature } from 'models'
```

Comment:
just double-checking, you're moving these over from common because we don't plan on using them on mobile?

Reply:
other way around. moving them over TO common because we DO plan on using them in mobile.